static-catalog/README.md

```diff
@@ -35,260 +59,181 @@ catalog (root)
 
-- `Catalog` -> `Item` (this is a common structure for a `catalog` to list links to `items`)
-- `Catalog` -> `Catalog` (this is a common tree structure to group sets of items. Each `catalog` in this relationship may also include `item` links as well as `catalog` links)
-- `Item` -> `Catalog` (example: an item may point to a catalog to describe a set of derived assets, where it may be desirable to have the origin asset as a "parent", such as NDVI generated from RGB/IR)
-- `Item` -> `Item` (example: this relationship may be used to describe a 1-1 parent-child relationship, such as a single derived item from one parent item)
+This example might be considered a somewhat "typical" structure. However, catalogs and items can
+describe a number of different relationships. The following shows various relationships between
+catalogs and items:
 
-The core `Item`s listed in a static catalog are the exact same form as those returned by a Catalog API.
-Ideally, STAC enables both a static and dynamic API to be crawled in the same way.
+- `Catalog` -> `Item` (this is a common structure for a catalog to list links to items)
+- `Catalog` -> `Catalog` (this is a common tree structure to group sets of items. Each catalog in
+  this relationship may also include item links as well as catalog links)
+- `Item` -> `Catalog` (example: an item may point to a catalog to describe a set of derived assets,
+  where it may be desirable to have the origin asset as a "parent", such as NDVI generated from
+  RGB/IR)
+- `Item` -> `Item` (example: this relationship may be used to describe a 1-1 parent-child
+  relationship, such as a single derived item from one parent item)
 
-An `Item` in STAC is compatible with a WFS 3.0 `Item`.
+The core Items listed in a static catalog are the exact same form as those returned by a Catalog
+API. Ideally, STAC enables both a static and dynamic API to be crawled in the same way.
 
-Catalogs make liberal use of the [rel](https://www.w3schools.com/tags/att_link_rel.asp) attribute on links to describe relationships, so that `Catalog`s and `Item`s always link to where they fit in and what else they are related to.
+Catalogs should be defined using the file name `catalog.json` to distinguish from item other JSON
+type files. In order to support multiple "root" catalogs, the recommended practice is to place the
+`catalog.json` in namespaces "directories". For example:
 
-### Catalog
+- current/catalog.json
+- archive/catalog.json
 
-A `Catalog` contains links out to `Item`s. Each of the `Catalog`s can link to `Item`s and other downstream `Catalog`s.
+A `catalog.html` is recommended with each `catalog.json` to make the catalog crawl-compatible.
+(Search engines do not recognize JSON as a crawlable data type)
 
-For example, a catalog of [NAIP imagery](https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/) might look something like this:
+### Static Catalog Flexibility
```

Comment:
The core spec starts to get pretty long here, and this & the next section don't necessarily add a whole lot. We could follow the stac-item folder a bit more closely, with a -spec.md and a readme. The readme can contain more info like this and the next section, and the -spec.md can be the actual core spec, and we keep that nice and tight.

Reply:
Sounds reasonable, maybe we should add that to #205.